BenGWeeks/Zapp.ie
# Bug 93844: Battery always shows as full

**Description**
I resolved two issues with this pull request: Battery always shows as full on Allowance tab and "Spent this week" section also on allowance tab that was not displaying correct number
Consulted with **Mario**.

**Screenshot/video**

![image](https://github.com/user-attachments/assets/fa421c86-f247-479b-ad13-935864a12cb4)
![image](https://github.com/user-attachments/assets/0b1f2740-61d8-4e33-9402-78e99899e7a2)

**Related workitems**
[[Bug 94788](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/94788): Merge Conflict: Incorrect SignOutButton Implementation](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/93844)
https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/96057

**Type of Change**

- [+]  Bug fix (non-breaking change which fixes an issue)

Test Instructions
To test this PR:
Go the the tabs folder with cd tabs
Run npm install
Run npm start
Log in
Open allowance  tab 
Take a look at how full the battery is
Check "Spent this week" value

Checklist:

- [+]  My code follows the style guidelines of this project (see "Contribute" in the wiki)
- [+]  I have performed a self - review of my own code
- [+]  My changes generate no new warnings
